data/.~lock.rtk.csv#

```diff
@@ -0,0 +1 @@
+,vrajur,Vinay-Precision-7510,24.07.2019 13:08,file:///home/vrajur/.config/libreoffice/4;
```

Comment:
This doesn't look like something you wanted to check in. Maybe tweak the .gitignore?

Reply:
Removed and updated .gitignore